src/MiniCover/HitServices/TestMethodInfo.cs

```diff
@@ -0,0 +1,110 @@
+﻿using MiniCover.Utils;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace MiniCover
+{
+    [Serializable]
+    public class TestMethodInfo : IEquatable<TestMethodInfo>
+    {
+        private static ConcurrentDictionary<string, bool> assemblyHasPdbCache = new ConcurrentDictionary<string, bool>();
+
+        public string AssemblyName { get; }
+        public string ClassName { get; }
+        public string MethodName { get; }
+        public string AssemblyLocation { get; }
+        public int Counter { get; private set; }
```

Comment:
I don't think we need to count how many times each test hited some instruction. That spend more time because of clones and so on.